apps/api/src/server/app/case-officer/__tests__/case-officer.get-appeals.test.js

```diff
@@ -0,0 +1,205 @@
+	id: 4,
+	reference: 'APP/Q8874/D/23/4293472',
+	appealStatus: [{
+		status: 'incomplete_lpa_questionnaire',
+		valid: true
+	}],
+	createdAt: new Date(2021, 12, 12),
+	addressId: 4,
+	startedAt: new Date(2022, 3, 20),
+	address: {
+		addressLine1: '180 Bloomfield Road',
+		postcode: 'BS4 3QX',
+		town: 'Bristol'
+	},
+};
+
+const appeal_5 = {
+	id: 5,
+	reference: 'APP/Q8874/D/23/1233427',
+	appealStatus: [{
+		status: 'available_for_statements',
+		valid: true },
+	{
+		status: 'incomplete_lpa_questionnaire',
+		valid: true }],
+	createdAt: new Date(2021, 12, 2),
+	addressId: 4,
+	startedAt: new Date(2022, 3, 2),
+	address: {
+		addressLine1: '30 Nowhere Road',
+		postcode: 'B36 TH2',
+		town: 'Brighton'
+	},
+};
+
+const appeal_6 = {
+	id: 6,
+	reference: 'APP/Q8874/D/23/7654321',
+	appealStatus: [{
+		status: 'available_for_statements',
+		valid: true },
+	{
+		status: 'available_for_final_comments',
+		valid: true
+	}],
+	createdAt: new Date(2021, 12, 2),
+	addressId: 4,
+	startedAt: new Date(2022, 3, 2),
+	address: {
+		addressLine1: '31 Somewhere Road',
+		postcode: 'B35 TH2',
+		town: 'Brighton'
+	},
+};
+
+const findManyStub = sinon.stub();
+findManyStub.returns([appeal_1, appeal_2, appeal_3, appeal_4, appeal_5, appeal_6]);
+
+class MockDatabaseClass {
+	constructor(_parameters) {
+		this.pool = {
+			appeal: {
+				findMany: findManyStub
+			}
+		};
+	}
+}
+
+test('gets the appeals information with received questionnaires', async (t) => {
+	sinon.stub(DatabaseFactory, 'getInstance').callsFake((arguments_) => new MockDatabaseClass(arguments_));
+	const resp = await request.get('/case-officer');
+	const appealExample = [{
+		AppealId : 1,
+		AppealReference: 'APP/Q9999/D/21/1345264',
+		QuestionnaireDueDate: '01 Jun 2022',
+		AppealSite: {
+			AddressLine1: '96 The Avenue',
+			AddressLine2: 'Maidstone',
+			County: 'Kent',
+			PostCode: 'MD21 5XY'
+		},
+		QuestionnaireStatus: 'awaiting'
+	},
+	{
+		AppealId : 2,
+		AppealReference: 'APP/Q9999/D/21/5463281',
+		QuestionnaireDueDate: '05 Jun 2022',
+		AppealSite: {
+			AddressLine1: '55 Butcher Street',
+			Town: 'Thurnscoe',
+			PostCode: 'S63 0RB'
+		},
+		QuestionnaireStatus: 'received'
+	},
+	{
+		AppealId: 3,
+		AppealReference: 'APP/Q9999/D/21/5463281',
+		AppealSite: {
+			AddressLine1: '55 Butcher Street',
+			Town: 'Thurnscoe',
+			PostCode: 'S63 0RB'
+		},
+		QuestionnaireDueDate: '05 Jun 2022',
+		QuestionnaireStatus: 'overdue',
+	},
+	{
+		AppealId: 4,
+		AppealReference: 'APP/Q8874/D/23/4293472',
+		AppealSite: {
+			AddressLine1: '180 Bloomfield Road',
+			Town: 'Bristol',
+			PostCode: 'BS4 3QX'
+		},
+		QuestionnaireDueDate: '04 May 2022',
+		QuestionnaireStatus: 'incomplete_lpa_questionnaire',
+	},
+	{
+		AppealId: 5,
+		AppealReference: 'APP/Q8874/D/23/1233427',
+		AppealSite: {
+			AddressLine1: '30 Nowhere Road',
+			PostCode: 'B36 TH2',
+			Town: 'Brighton',
+		},
+		QuestionnaireDueDate: '16 Apr 2022',
+		QuestionnaireStatus: 'incomplete_lpa_questionnaire',
+	},
```

Comment:
this one should appear twice in the array. once with `QuestionnaireStatus: 'incomplete_lpa_questionnaire'` and once with `StatementsAndFinalCommentsStatus: 'available_for_statements'`

Reply:
New specific format for parallel status added